about.html

```diff
@@ -0,0 +1,119 @@
+                </ul>
+                <ul class="nav-link">
+                    <li class="home-link"><a href="./">Home</a></li>
+                    <li><a class="active" href="./about.html">About</a></li>
+                    <li><a href="#program">Program</a></li>
+                    <li><a href="#speaker">Speakers</a></li>
+                    <li><a href="#contact">Sponsor</a></li>
+                    <li><a href="#contact">News</a></li>
+                    <li><a href="#" class="btn btn-outline-primary">Campaign</a></li>
+                </ul>
+            </section>
+        </nav>
+        <section class="hero container text-center">
+            <p class="greeting">"Hello! Sharing world"</p>
+            <h1>MV FULL-STACK <br> GRADUATION EVENT 2023 </h1>
+            <div class="description">
+                <p>The Global Event brings together the community of professional, and activists who comprise the Spiritual World affiliate network in a different region every year. </p><p class="desc-desktop">This year, we hope to expand our invitation list including organizations and individuals. who want to participate and shared their experiences.</p><p class="desc-desktop">Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi libero at optio placeat qui quia veniam eius deleniti. Impedit vitae incidunt delectus aspernatur tempore pariatur expedita minus sunt natus quo.</p>
+            </div>
+            <p class="location">Please contact us per Email for any further questions about MV Fullstack Graduation Event 2023!</p>
+
+            <a class="hero-mail" href="gmail.com">swevent2023@swmyanmar.org <i class="fa-solid fa-chevron-right"></i></a>
+        </section>
+    </header>
+    <main>
+        <section class="event-logo">
+            <div class="container">
+                <div class="heading">
+                    <h2 class="heading-primary">MV Full-Stack Graduation 2023 Logo</h2>
+                    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Earum, similique voluptatem.</p>
+                </div>
+                <div class="logo-section text-center">
+                    <img src="assets/images/icon.svg" alt="MV logo">
+                </div>
+            </div>
+        </section>
+
+        <section class="past-events">
+            <div class="container">
+                <div class="heading">
+                    <h2 class="heading-primary">See the past MV Full-Stack Graduations</h2>
+                    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Earum, similique voluptatem.</p>
+                </div>
+                <div class="past-event text-center">
+                    <img class="img" src="assets/images/2021.png" alt="2021 event">
+                    <img class="img" src="assets/images/2022.png" alt="2022 event">
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <footer>
+        <section class="partners">
```

Comment:
- [ ] I would like to recommend you move this section inside your main tag for a semantic reason. Please do it for your both index and about pages :+1: